edb/server/protocol/protocol.pyx

```diff
@@ -574,7 +574,7 @@ cdef class HttpProtocol:
                 request,
                 response,
                 path_parts[1:],
-                self.server,
+                self.tenant,
```

Comment:
Should use `self._ensure_tenant()` here, fixed in [a future commit](https://github.com/edgedb/edgedb/pull/5742/commits/674979a667c96533353058707cd0d62b8b8a950c)